recipes/hyperspy_gui_traitsui/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hyperspy-gui-traitsui" %}
+{% set filename = "hyperspy_gui_traitsui" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "a2c8ca1917babee6c604a14988dc7907e8d13581c6307863e71d0cab05d27a74" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ filename }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - qt 4.8.*
+    - traitsui >=5.0
+    - hyperspy >=1.3
+
+test:
+  commands:
+    - export MPLBACKEND="agg"  # [unix]
+    - py.test --pyargs hyperspy_gui_traitsui
+
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/hyperspy/hyperspy_gui_traitsui
+  license: GPL v3
+  summary: "traitsui GUI elements for HyperSpy."
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Please ask upstream to add the license file to `MANIFEST.in` for future inclusion with the PyPI `sdist`.